docs: update readme instructions regarding nextclade full run (#230)
nextstrain/ncov-ingest: README.md

The most convenient option is to trigger it through the corresponding GitHub Action:

 - [GISAID full Nextclade run](https://github.com/nextstrain/ncov-ingest/actions/workflows/nextclade-full-run-gisaid.yml)
 - [GenBank full Nextclade run](https://github.com/nextstrain/ncov-ingest/actions/workflows/nextclade-full-run-genbank.yml)

They will simply run the `./bin/run-nextclade-full-aws --database=<name of the database>` and will announce the beginning of the job and the AWS Batch Job ID on Nextstrain Slack. 

For that, go to the GitHub Actions UI using one of the links above, click the button "Run workflow", choosing "branch: master" from the list and confirming.

If needed, the runs can be also launched from a local machine, by one of these scripts, depending on whether you want to run the computation locally, in docker, or to schedule an AWS Batch Job (the latter is what GitHub Actions do):

```bash
./bin/run-nextclade-full           # Runs locally (requires significant computational resources)
./bin/run-nextclade-full-aws       # Runs in docker (requires significant computational resources)
./bin/run-nextclade-full-docker    # Schedules an AWS Batch Job and runs there
```

> In case of AWS Batch option, the results of the computation, the new `nextclade.tsv` will be uploaded to S3 into a subdirectory in the directory which is the usual location of this file for the database. The subdirectory name will contain a date, so that there is no confusion about versions. The Slack announcement will contain the full path. These files then need to be manually inspected for correctness and scientific soundness and id all good, copied to the usual location where the daily ingest can find them. From that point the clades are considered fresh.

Reviewer: ```suggestion
With the AWS Batch option, the results of the computation – the new `nextclade.tsv` – will be uploaded to S3 into a subdirectory of the directory which is the usual location of this file for the database. The subdirectory name will contain a date, so that there is no confusion about versions. The Slack announcement will contain the full path. These files then need to be manually inspected for correctness and scientific soundness and if all good, copied to the usual location where the daily ingest can find them. From that point the clades are considered fresh.
```

Author: Thank you, sir :)
